commit a55b59fef35af3fa86d6b6279f92ca2e3a124252
Author: Kristian Rietveld <kris gtk org>
Date:   Thu Mar 10 21:20:37 2011 +0100

    Include gdkquartz.h so GDK_IS_QUARTZ_SCREEN is actually defined

 gtk/gtksettings.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)
---
diff --git a/gtk/gtksettings.c b/gtk/gtksettings.c
index d870b44..3a83440 100644
--- a/gtk/gtksettings.c
+++ b/gtk/gtksettings.c
@@ -39,6 +39,10 @@
 #include <pango/pangofc-fontmap.h>
 #endif
 
+#ifdef GDK_WINDOWING_QUARTZ
+#include "quartz/gdkquartz.h"
+#endif
+
 
 /**
  * SECTION:gtksettings

